specifyweb/stored_queries/batch_edit.py

```diff
@@ -406,7 +406,7 @@ def _recur_row_plan(
 
         rel_type = (
             "to_many"
-            if node.type.endswith("to-many") or node.type == "zero-to-one"
+            if node.type.endswith("to-many") or node.type == "zero-to-one" or node.is_remote_to_one()
```

Comment:
This took care of the upload plan changes needed for remote to-ones. However, changes made to COJO records don't get updated when committing. Changes made to a COJO record's `childCO` or `childCog` work fine too.